v0/destinations/posthog/data/PHGroupConfig.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "destKey": "timestamp",
+    "sourceKeys": ["originalTimestamp", "timestamp"],
+    "required": false
+  },
+  { "destKey": "groupId", "sourceKeys": "groupId", "required": false },
+  { "destKey": "context", "sourceKeys": "context", "required": false },
+  {
+    "destKey": "distinct_id",
+    "sourceKeys": [
+      "userId",
+      "context.traits.userId",
+      "context.traits.id",
+      "anonymousId"
+    ],
+    "required": true
+  },
+  { "destKey": "traits", "sourceKeys": "context.traits", "required": false },
```

Comment:
check the location of group traits in rudder payload. Confirm for web/mobile/server sources.